src/pages/guidelines/ibm-dot-com-style-models/overview.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: IBM.com style models
+description: IBM.com style models provide a design framework that defines the foundational and structural elements of IBM.com pages in order to establish consistency and coherency across users' journeys in a scalable way.
+tabs: ['Overview', 'Style model list']
+---
+<PageDescription>
+
+IBM.com style models provide a design framework that defines the foundational and structural elements of IBM.com pages in order to establish consistency and coherency across users&rsquo; journeys in a scalable way.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Our vision</AnchorLink>
+<AnchorLink>Style model</AnchorLink>
+<AnchorLink>Using style models</AnchorLink>
```

Comment:
Should Style be capital?

Reply:
Let's keep it in all lower case for now.